src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -12,37 +13,43 @@
           toggled-state (if (= :on flashlight-state) :off :on)]
       (assoc-in db [:wallet :send-transaction :camera-flashlight] toggled-state))))
 
-(defn- fill-request-details [db address name amount]
+(defn- fill-request-details [db {:keys [address name value symbol gas gasPrice] :as m}]
+  {:pre [(and (not (nil? address)) (not (nil? name)))]}
```

Comment:
Wait, what about the other parameters? What happens if they are nil?

Reply:
They are ignored as this is guarded by the `cond-> `